docker.go

```diff
@@ -53,9 +53,12 @@ func (docker *dockerConfig) call() int {
 	const logLevelArg = "--terragrunt-logging-level"
 	if !listContainsElement(command, logLevelArg) && filepath.Base(config.EntryPoint) == "terragrunt" {
 		level, _ := strconv.Atoi(config.LogLevel)
-		if level > 6 || strings.ToLower(config.LogLevel) == "full" {
-			config.LogLevel = "trace"
+		if strings.ToLower(config.LogLevel) == "full" {
```

Comment:
hmm didn't know about that logging level

Reply:
I think the goal was only to activate the Terraform debug (which is very verbose). But that should not be related to the actual supplied trace level.